_posts/2021-03-29-Encoding the Lending Library.md

```diff
@@ -0,0 +1,214 @@
+
+<!--more-->
+
+### Ellen Wright
+Through the Shakespeare and Company Project website, I selected the collection of [all cards associated with Ellen Wright](https://shakespeareandco.princeton.edu/members/wright-ellen/cards/0d89fdbb-a48a-4bfa-8690-c53234d02331/), an American living in Paris with her husband, the acclaimed African American author Richard Wright. There are two cards associated with Wright, pictured below:
+
+![Front of Card 1](https://iiif.princeton.edu/loris/figgy_prod/13%2Fe8%2F39%2F13e839f94ac344f7bc0a5793c5c4df80%2Fintermediate_file.jp2/full/450,/0/default.jpg)
+
+![Front of Card 2](https://iiif.princeton.edu/loris/figgy_prod/93%2Fdd%2F4d%2F93dd4d3a5fa048ed8e88685bd2df1e0c%2Fintermediate_file.jp2/full/450,/0/default.jpg)
+
+###### Image Credits: [Wright, Ellen; Sylvia Beach Papers, C0108, Manuscripts Division, Department of Special Collections, Princeton University Library](https://findingaids.princeton.edu/catalog/C0108_c3762)
+
+Though there is overlap between the information on the two cards, I chose to encode them both. I did not encode the backs of the cards, as the first card is blank on the reverse, and the other side of second card is an advertisement from the Princeton Bank and Trust Company.
+
+### Important Elements
+When encoding these cards, I began by identifying the key items that formed the basis of the borrowing records: the borrow date, the book title, and the return date. I also encoded the heading on the top of each card, which contained the library member's name and home address. Another key aspect of these cards were the other library members mentioned, many of whom were accomplished authors themselves, such as Richard Wright and Gertrude Stein. Finally, I noted that the handwritten text on each card changed color multiple times, a result of the process by which the card was updated over the span of months each time Wright borrowed or returned books. Though the particular colors used may not have much significance, they provide evidence for the ways the card changed over time. By capturing the key information about an individual's borrowing habits in this encoded form, it is possible to analyze this data more effectively and systematically compare Wright's records with those of other library members.
+
+I used the transcriptions of book titles and dates on the Shakespeare and Company Project database to confirm my reading of the cards. However, as I wanted to encode the card in full, I also had to parse the notes and other markings on each card.
+
+### The XML Encoding
+
+``` <div type='all-lending-library-cards'>
+	<div type="lending-library-card">
+		<side type="front">
+			<heading>
+				<text color="blue">
+					<member-name>Mrs Richard WRIGHT</member-name>
+					<address>14 rue M. le Prince, Paris VI</address>
+				</text>
+			</heading>
+			<borrowing-record>
+				<text color="blue">
+					<date-start>Sept 19 1949</date-start>
+					<title>The Pasquier Chronicles</title>
```

Comment:
Another thought about structuring this data: if, for example, a researcher wanted to know when and for how long once of these titles was lended out, as the data is structured here these dates and titles are encoded as sibling elements under the parent item `<borrowing-record>` and `<text color>`, which means there is no relationship between these dates and the titles they accompany. Because of the ongoing use of these cards, there are gaps between when a book was lent and when it was returned, which makes your XML structure tricky to work out, I know. As this card is encoded now, the `<text>` tag prevents you from linking the return date of _Walter Rathenau_ to its lending date. I love that you have noted text color but if we were embarking on a project of encoding all of the lending library, I think this would become a big debate for us as researchers: how to denote the physical features of the card while also demonstrating how these books move in and out of the library over time.